vipps-recurring-api-faq.md

```diff
@@ -26,7 +26,7 @@ Document version: 1.4.2.
 - [When can I send charges for a user?](#when-can-i-send-charges-for-a-user)
 - [How can I delete an agreement?](#how-can-i-delete-an-agreement)
 - [Can a user cancel the agreement through the Vipps app?](#can-a-user-cancel-the-agreement-through-the-vipps-app)
-- [What happens to charges if the user cancels the agreement?](#what-happens-to-charges-if-the-user-cancels-the-agreement)
+- [What happens to charges if the corresponding agreement is cancelled?](#what-happens-to-charges-if-the-corresponding-agreement-is-cancelled)
```

Comment:
I think it's nice to explicitly say who cancels: Is it the merchant or the user? Avoiding ambiguity is A Good Thing.

Reply:
It's always the merchant doing it though.
The user asks the merchant to cancel the agreement using whatever interface the merchant provides to said user